web/themes/custom/usagov/usagov.theme

```diff
@@ -444,3 +444,65 @@ function usagov_theme_suggestions_alter(array &$suggestions, array $variables, $
     }
   }
 }
+
+function usagov_preprocess_breadcrumb(&$variables) {
+
+  $database = \Drupal::database();
+
+  if (strpos($_SERVER['REQUEST_URI'], 'agencies/') !== FALSE) {
+    $variables['breadcrumb'] = [];
+    $variables['breadcrumb'][] = ['text' => 'Home', 'url' => 'https://www.usa.gov/'];
+    $nid = $database->query("SELECT path FROM path_alias WHERE alias LIKE '%about-the-us%'")->fetchField();
+    $nid = str_replace('/node/', '', $nid);
+    $title = $database->query("SELECT title FROM node_field_data WHERE nid = {$nid}")->fetchField();
+    $variables['breadcrumb'][] = ['text' => $title, 'url' => '/about-the-us'];
+    $nid = $database->query("SELECT path FROM path_alias WHERE alias LIKE '%agency-index%'")->fetchField();
```

Comment:
same as above but for Spanish, you might have to include the langcode as the second parameter to `getPathByAlias()`